test/framework/src/main/java/org/elasticsearch/index/mapper/TestBlock.java

```diff
@@ -57,6 +57,18 @@ public BytesRefsBuilder appendBytesRef(BytesRef value) {
                 return new BytesRefsBuilder();
             }
 
+            @Override
+            public BlockLoader.BytesRefBuilder geometries(int expectedCount) {
+                class GeometriesBuilder extends TestBlock.Builder implements BlockLoader.BytesRefBuilder {
+                    @Override
+                    public GeometriesBuilder appendBytesRef(BytesRef value) {
+                        add(BytesRef.deepCopyOf(value));
```

Comment:
This *looks* like we can just use `bytesRefs` above.

I think we needed this when we had a totally different block type - one that natively supported points. But we don't so much now.

Reply:
Removed now, along with the simplifications above